### ISSUE TITLE
Poor accuracy detecting eyebrow landmarks.

### ISSUE THREAD

**Issue by Ailuridae** [davisking/dlib#1658]
The pre-trained 68 landmark shape predictor is very good at detecting the overall shape and orientation. The mouth and eyes are usually detected correctly as well. However, it somehow does not track eyebrow movement at all. That is, raised eyebrows are not detected at all. It does seem to somewhat accurately detect lowered eyebrows.

I've already tried training new predictors using varying parameters, as well as limiting the landmark set to the left and right eyebrows respectively. I made sure the training set contained a reasonable amount of example images. 

What could cause the poor accuracy? Would training the predictor using a larger bounding box result in better performance? Sometimes, raised eyebrows are not within the detected bounding box.

**Comment by davisking** [comment]
This is mainly because the training dataset doesn't include a lot of people moving their eyebrows up.  The way to fix this is to get more and better training data.  

Being outside the box doesn't really matter.  For a detailed understanding read the manual: http://dlib.net/dlib/image_processing/shape_predictor_trainer_abstract.h.html#shape_predictor_trainer

**Comment by Ailuridae** [comment]
Thanks. I made sure the training had enough examples, but the difference in appearance between the training and testing set was too large. I was able to solve the problem by using a more appropriate combination.
